site/en/workflows.md

```diff
@@ -671,9 +695,12 @@ A line detection algorithm is run on every text region of every PAGE in the
 input file group, and a TextLine element with the resulting polygon
 outline is added to the annotation of the output PAGE.
 
+**Note:** If you use `ocrd-cis-ocropy-segment`, you can directly go on with [Step 13](#step-13-dewarping-on-line-level).
+
 **Note:** If you use `ocrd-tesserocr-segment-line`, which uses only bounding boxes instead of polygon coordinates,
-then you should post-process with the processors described in [Step 12](#step-12-resegmentation-line-level).
-If you use `ocrd-cis-ocropy-segment`, you can directly go on with [Step 13](#step-13-dewarping-on-line-level).
+then you should post-process with the processors described in [Step 12](#step-12-resegmentation-line-level). Alternatively,
+you can use the [`ocrd-tesserocr-recognize` all-in-one processor](#step-x-multistep) that can do line segmentation, region segmentation
+and recognition in one step.
```

Comment:
```suggestion
then you should post-process with the processors described in [Step 12](#step-12-resegmentation-line-level). 
_Alternatively_, consider using the all-in-one capabilities of [`ocrd-tesserocr-recognize`](#step-x-multistep), which can do line segmentation and text recognition in one step by querying Tesseract's internal iterator (accessing the more precise polygon outlines instead of just coarse bounding boxes with lots of hard-to-recover overlap).
```